dataingest/spatial/README.md

```diff
@@ -0,0 +1,16 @@
+* AWS
+https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.PostgreSQL.CommonDBATasks.PostGIS.html
+
+Location of GeoJSON download
+
+* LAD
+
+https://geoportal.statistics.gov.uk/datasets/ons::local-authority-districts-december-2017-boundaries-in-the-uk-wgs84/explore?layer=3&location=54.650000%2C-3.250000%2C5.66
```

Comment:
might be worth shortening including the direct download link for the geojson as well? `https://geoportal.statistics.gov.uk/datasets/ons::local-authority-districts-december-2017-boundaries-in-the-uk-wgs84/explore?layer=3&location=54.145542%2C-3.250000%2C5.66#:~:text=Generate%20new%20download-,with,-latest%20data`

Reply:
Yes many of these links aren't curl-able since they use Javascript. I'm tempted to check in compressed versions but don't want to bloat the repo. 

Maybe I should check into "ONSdigital/fi-census-data" so at least we have copies.

Reply:
I guess at least that way we can point at it and say, thats the data we used

Reply:
OOS for this though!